pom.xml

```diff
@@ -127,6 +127,7 @@
                         <arg>-unchecked</arg>
                         <arg>-Xmax-classfile-name</arg>
                         <arg>140</arg>
+                        <arg>-nobootcp</arg>
```

Comment:
what does this do?

Reply:
This removes the scala jar from the boot classpath and is need to compile scala code on JDK9+